apps/experiments/views/experiment.py

```diff
@@ -448,10 +448,10 @@ def single_experiment_home(request, team_slug: str, experiment_id: int):
     )
     channels = experiment.experimentchannel_set.exclude(platform__in=[ChannelPlatform.WEB, ChannelPlatform.API]).all()
     used_platforms = {channel.platform_enum for channel in channels}
-    available_platforms = set(ChannelPlatform.for_dropdown(experiment.team)) - used_platforms
+    available_platforms = ChannelPlatform.for_dropdown(used_platforms, experiment.team)
     platform_forms = {}
     form_kwargs = {"team": request.team}
-    for platform in available_platforms:
+    for platform in available_platforms.keys():
```

Comment:
no need to use `.keys()`, iterating over a dict will iterate over the keys already

Reply:
[changed](https://github.com/dimagi/open-chat-studio/pull/572/commits/cb0498ef7d72b9e20e252eed6e14d805ad1dd0a0)